build.gradle.kts

```diff
@@ -140,6 +142,31 @@ subprojects {
 		}
 
 		afterEvaluate {
+			if (project != project(":junit-jupiter")) {
```

Comment:
Please add this to `java-library-conventions.gradle.kts` instead of to the root build script.

Reply:
Great, I wasn't aware of it's existence.

Reply:
I managed to move the bnd configuration over to `java-library-conventions.gradle.kts`